pkg/core/native/designate.go

```diff
@@ -121,7 +121,15 @@ func newDesignate(initialNodeRoles map[noderoles.Role]keys.PublicKeys) *Designat
 	eDesc := newEventDescriptor(DesignationEventName,
 		manifest.NewParameter("Role", smartcontract.IntegerType),
 		manifest.NewParameter("BlockIndex", smartcontract.IntegerType))
-	eMD := newEvent(eDesc)
+	eMD := newEvent(eDesc, config.HFDefault, config.HFEchidna)
```

Comment:
>  native: add Designation event extension after HFEchidna 

Why `after`? It's valid starting from Echidna, so it's either `to` or `starting from`.